src/PaymentMethods/GiftCardPaymentMethod.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Adyen\Shopware\PaymentMethods;
+
+use Adyen\Shopware\Handlers\GiftCardPaymentMethodHandler;
+
+class GiftCardPaymentMethod implements \Adyen\Shopware\PaymentMethods\PaymentMethodInterface
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function getName(): string
+    {
+        return 'GiftCard';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getDescription(): string
+    {
+        return 'GiftCard';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getPaymentHandler(): string
+    {
+        return GiftCardPaymentMethodHandler::class;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getGatewayCode(): string
+    {
+        return 'ADYEN_GIFTCARD';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getTemplate(): ?string
+    {
+        return null;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getLogo(): string
+    {
+        return 'giftcard.png';
```

Comment:
Do we have this file in the checkoutshopper remote directory?

Reply:
Okay we do, nevermind 😄
https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/giftcard.png